### ISSUE TITLE
When runing ./debug, It's show xterm:Command not found

### ISSUE THREAD

**Issue by sususama** [ControlCore-Project/concore-lite#3]
Recurring current issues:
- $ git clone https://github.com/ControlCore-Project/concore-lite.git
- $ cd concore-lite
- $ ./makestudy example/sampleplot.graphml
- $ cd sampleplot/
- $ ./build
- $ ./debug
![1679212220458](https://user-images.githubusercontent.com/52159774/226161375-c1e504d9-2882-4ae6-8201-5ce639e75969.png)
When I try to install Xterm:
![1679212324296](https://user-images.githubusercontent.com/52159774/226161459-f5af987c-dc12-4d85-ac45-948d7b3787c9.png)

**Comment by pradeeban** [comment]
Yes, debug command pops up windows. You should have xterm installed (which is usually already installed in Linux local installations, but not server versions).

For server execuctions, debug won't work (since you cannot open/show windows/popups remotely anyway.

You can use "run" instead.

This is not a bug.
